python/lsst/daf/butler/formatters/parquet.py

```diff
@@ -89,13 +90,15 @@ def _splitColumnnNames(n: int, names: Iterable[str]) -> Iterator[Tuple[str, ...]
         tuple : `tuple` of `str`
             A multi-index column name tuple.
         """
-        pattern = re.compile(r"\({}\)".format(', '.join(["'(.*)'"] * n)))
+        pattern = re.compile(r"\({}\)".format(", ".join(["'(.*)'"] * n)))
         for name in names:
             m = re.search(pattern, name)
             if m is not None:
                 yield m.groups()
 
-    def _standardizeColumnParameter(self, columns: Dict[str, Union[str, List[str]]]) -> Iterator[str]:
+    def _standardizeColumnParameter(
+        self, columns: Union[List[tuple], Dict[str, Union[str, List[str]]]]
+    ) -> Iterator[str]:
```

Comment:
I don't know if this passed PEP8 but I find it really hard to read and understand.

Reply:
This is just adding an option for the `columns` keyword to be passed a list of tuples instead of just a dictionary of a particular format, which was the original implementation.